packages/runtime/src/dataViews/DataViewTranslateable.ts

```diff
@@ -7,31 +7,18 @@ export class DataViewTranslateable {
         relationshipRejected: `${DataViewTranslateable.prefix}relationship.Rejected`,
         relationshipRevoked: `${DataViewTranslateable.prefix}relationship.Revoked`,
         relationshipActive: `${DataViewTranslateable.prefix}relationship.Active`,
+        relationshipTerminated: `${DataViewTranslateable.prefix}relationship.Terminated`,
+        relationshipDeletionProposed: `${DataViewTranslateable.prefix}relationship.DeletionProposed`,
         fileName: `${DataViewTranslateable.prefix}file.name`
     };
 
     public static readonly consumption = {
         mails: {
-            mailSubjectFallback: `${DataViewTranslateable.prefix}mails.mailSubjectFallback`,
-            requestMailSubjectFallback: `${DataViewTranslateable.prefix}mails.requestMailSubjectFallback`
+            mailSubjectFallback: `${DataViewTranslateable.prefix}mails.mailSubjectFallback`
         },
         attributes: {
             unknownAttributeName: `${DataViewTranslateable.prefix}attributes.UnknownAttributeName`
         },
-        requests: {
-            // Request for sharing an attribute
-            attributesShareRequestName: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.name`,
-            // Request for sharing multiple attributes
-            attributesShareRequestNamePlural: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.namePlural`,
-            // You don't have a relationship to any of the recipients of this request
-            attributesShareRequestNoRelationship: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.noRelationship`,
-            // The data is only shared with known recipients
-            attributesShareRequestOnlyRelationships: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.onlyRelationships`,
-            // Request for changing an attribute
-            attributesChangeRequestName: `${DataViewTranslateable.prefix}requests.AttributesChangeRequest.name`,
-            // Request for changing multiple attributes
-            attributesChangeRequestNamePlural: `${DataViewTranslateable.prefix}requests.AttributesChangeRequest.namePlural`
-        },
```

Comment:
why is this removed?

Reply:
From the Announcing enmeshed v2 requests blog: The `RequestMail`, `AttributesChangeRequest` and `AttributesShareRequest` types are now deprecated and will be removed in the future.

I just went ahead with it.

Reply:
oh, I see!